### ISSUE TITLE
Keyboard focus is not visible on the elements while navigating with tab key.

### ISSUE THREAD

**Issue by AgneLukoseviciute** [microsoft/react-native-gallery#222]
Found during accessibility grading.

Tracking internal bug: https://microsoft.visualstudio.com/DefaultCollection/OS/_workitems/edit/40125050

**Comment by jonthysell** [comment]
It looks like some elements (like hyperlinks) have no obvious focus rect, so it's easy to get lost tabbing and not get to the items on the page that you're looking for. It's also possible there are some non-interactive items grabbing tab focus that shouldn't be.

**Comment by AgneLukoseviciute** [comment]
From ADO issue: "focus will be visible after hitting tab key for so many times" -- sounds a lot like react native screens not unmounting, will try using a hook to disable controls from the previous page(s) as a workaround.

**Comment by AgneLukoseviciute** [comment]
+noticed just now that the items in the navigation pane itself are also part of the tab index, even if the navpane is not expanded. So keyboard focus seems to disappear while you tab through the whole 34 items in the navpane.

**Comment by AgneLukoseviciute** [comment]
Reopening, I see a different issue that this might be referring to & which is more apparent now that the above PR is merged. It looks as though in (some?) example pages, all of the elements are tabbed through once or twice in a row without any visible keyboard focus. If you turn on narrator, you can hear the elements being announced, however no keyboard focus is visible until eventually one of the iterations through a page keyboard/narrator focus appears. Maybe some duplication of the xaml tree happening.

**Comment by AgneLukoseviciute** [comment]
Update: as of commit ad046238a8d4e465b8aa2053842ed4e6749c9084, this was still the case so this wasn't an unintended side effect of any of the recent a11y work. Looking from accessibility insights you can see that there are actually n instances of each page, with n seeming to correlate with how many pages you had previously navigated to -- suggests that this at least in part will be related to react-native screens: https://github.com/microsoft/react-native-windows/issues/4152

**Comment by AgneLukoseviciute** [comment]
This will be fixed by your change Chiara, so reassigning.
